open-reac/src/test/java/com/powsybl/openreac/OpenReacRunnerTest.java

```diff
@@ -78,6 +78,7 @@ void test() throws IOException {
             OpenReacResult openReacResult = OpenReacRunner.run(network, network.getVariantManager().getWorkingVariantId(), parameters, config, computationManager);
 
             assertEquals(OpenReacStatus.OK, openReacResult.getStatus());
+            assertEquals(34, openReacResult.getGeneratorModifications().size());
```

Comment:
Do you have updated the output files with last version of AMPL OpenReac (if is has changed of course)?

Reply:
The modifs output are a bit too low...